build/tools/amebalite/gnu_utility/AP2_SWDPWD.JLinkScript

```diff
@@ -0,0 +1,240 @@
+/*********************************************************************
```

Comment:
Let's move the change in build into 4th commit (change in build folder including build/configs)